tests/utils.py

```diff
@@ -817,3 +817,20 @@ def load_nist_kbkdf_vectors(vector_data):
             test_data[name.lower()] = value.encode("ascii")
 
     return vectors
+
+
+def load_ed25519_vectors(vector_data):
+    data = []
+    for line in vector_data:
+        x = line.split(':')
+        secret_key = x[0][0:64]
```

Comment:
The info is in the initial PR comment :smile:

Here's what the indices are:

0: secret key (32 bytes) + public key (32 bytes)
1: public key (32 bytes)
2: message (0+ bytes)
3: signature (64 bytes) + message (0+ bytes)

So we're slicing because djb's vectors are a bit weird and concatenate info in ways we don't need/want.